doc/api/globals.md

```diff
@@ -637,6 +637,21 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.platform`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.platform` read-only property returns a string identifying the
+platform on which the Node.js instance is running.
```

Comment:
This may or may not be portable. MDN strongly discourages the use of `navigator.platform` and Deno doesn't implement it AFAIK. Browsers may return almost arbitrary values for the sake of privacy. Perhaps add a note discouraging users from using it. (Then again, the same can be said for most of `navigator`.)

Reply:
It is not spec compliant to return an empty string. See https://github.com/whatwg/html/pull/7762